pycbc/waveform/bank.py

```diff
@@ -178,6 +179,26 @@ def parse_approximant_arg(approximant_arg, warray):
     """
     return warray.parse_boolargs(boolargs_from_apprxstr(approximant_arg))[0]
 
+def list_to_hash(list_to_be_hashed):
```

Comment:
To avoid confusion, I don't think you should call this 'list_XXX'. Hashing a list doesn't actually work. It really does have to be a tuple.